packages/cheqd-sdk/src/anoncreds/utils/transform.ts

```diff
@@ -0,0 +1,255 @@
+// To parse this data:
+//
+//   import { Convert, CheqdSchema, CheqdCredentialDefinition, CheqdRevocationRegistryDefinition } from './file'
+//
+//   const cheqdSchema = Convert.toCheqdSchema(json)
+//   const cheqdCredentialDefinition = Convert.toCheqdCredentialDefinition(json)
+//   const cheqdRevocationRegistryDefinition = Convert.toCheqdRevocationRegistryDefinition(json)
+//
+// These functions will throw an error if the JSON doesn't
+// match the expected interface, even if the JSON is valid.
+
+export interface CheqdSchema {
+  name: string
+  attrNames: string[]
+  version: string
+}
+
+export interface CheqdCredentialDefinition {
+  schemaId: string
+  type: 'CL'
+  tag: string
+  value: CheqdCredentialDefinitionValue
+}
+
+export interface CheqdCredentialDefinitionValue {
+  primary: { [key: string]: any }
+  revocation?: any
+}
+
+export interface CheqdRevocationRegistryDefinition {
+  revocDefType: 'CL_ACCUM'
+  credDefId: string
+  tag: string
+  value: CheqdRevocationRegistryDefinitionValue
+}
+
+export interface CheqdRevocationRegistryDefinitionValue {
+  publicKeys: PublicKeys
+  maxCredNum: number
+  tailsLocation: string
+  tailsHash: string
+}
+
+export interface PublicKeys {
+  accumKey: AccumKey
+}
+
+export interface AccumKey {
+  z: string
+}
+
+// Converts JSON strings to/from your types
+// and asserts the results of JSON.parse at runtime
+export class Convert {
+  public static toCheqdSchema(json: string): CheqdSchema {
```

Comment:
I see you call `json.stringify` before passing it to this method, and then calling `json.parse` here again. Maybe we can save some resource by passing in the json directly and not calling `json.parse` here?